tests/e2e/provision_with_multiple_zones.go

```diff
@@ -158,4 +159,64 @@ var _ = ginkgo.Describe("[csi-topology-vanilla] Topology-Aware-Provisioning-With
 		pv = nil
 
 	})
+
+	/*
+		Provisioning with multiple zones and regions specified in the Storage Class for non-shared datastore
+		//Steps
+		1. Create SC with multiple Zone and region details specified in the SC
+		2. Create statefulset with replica 5 using the above SC
+		3. Wait for PV, PVC to bound
+		4. Statefulset should get distributed across zones.
+		5. Describe PV and verify node affinity details should contain both zone and region details
+		5a. If a volume provisioned on datastore that is shared only within zone1 then node affinity will have details of only zone1
+		5b. If a volume provisioned on datastore that is shared only on zone2 then node affinity will have details of only zone2
+		6. Verify statefulset pod is running on the same node as mentioned in node affinity details
+		7. Delete POD,PVC,PV
+	*/
+	ginkgo.It("Verify provisioning with multiple regions and zones with different cluster datastore", func() {
```

Comment:
Provisioning volume using storage policy with multiple zone and region details in the allowed topology

Reply:
done